build.gradle.kts

```diff
@@ -1,3 +1,6 @@
+import com.adarshr.gradle.testlogger.theme.ThemeType
```

Comment:
Is it possible to use the documented Gradle support instead of this custom code? I'd like to understand the differences.

https://appmap.io/docs/reference/appmap-gradle-plugin.html#installation

Reply:
Let me try that. I didn't know that there's a AppMap Gradle  plugin

Reply:
I made a change for this at https://github.com/getappmap/appmap-intellij-plugin/pull/374, but had to drop the commit from this branch because the AppMap Gradle plugin is currently breaking our build and tests: https://github.com/getappmap/appmap-gradle-plugin/issues/18